ixmp4/data/db/docs.py

```diff
@@ -60,20 +62,29 @@ def select(
         *,
         _exc: db.sql.Select[tuple[DocsType]] | None = None,
         dimension_id: int | None = None,
+        dimension_id__in: Iterable[int] | None = None,
     ) -> db.sql.Select[tuple[DocsType]]:
         if _exc is None:
             _exc = db.select(self.model_class)
 
+        # TODO Should we enforce that only one of these filters can be active?
         if dimension_id is not None:
             _exc = _exc.where(self.model_class.dimension__id == dimension_id)
+        elif dimension_id__in is not None:
```

Comment:
```suggestion
        if dimension_id__in is not None:
```
This might seem kind of unnecessary, but it makes sure that the parameter is not just quietly ignored when `dimension_id` is supplied.